azkaban-common/src/main/java/azkaban/executor/Status.java

```diff
@@ -82,6 +85,7 @@ public static boolean isStatusRunning(final Status status) {
 
   public static boolean isStatusFailed(final Status status) {
     switch (status) {
+      case POD_FAILED:
```

Comment:
In conditional flows, this method is used to check if we need to run other nodes. when we combine the node failures with system-related failure (in this case pod), this will likely affect that feature. It's worth double-checking that conditional flow use case.